components/webui/common/tsconfig.json

```
@@ -0,0 +1,32 @@
+{
+  "compilerOptions": {
+    "esModuleInterop": true,
+    "lib": [
+      "es2023", "dom"
```

to be consistent with the casing documented at https://www.typescriptlang.org/tsconfig/#high-level-libraries
```suggestion
      "DOM"
      "ES2023", 
```